requirements.txt

```diff
@@ -3,7 +3,7 @@ PyYAML
 requests>=2.0
 requests-mock>=0.7,<0.8
 django-crispy-forms>=1.6,<1.7
-django-selectable>=0.9.0,<1.0
+git+https://github.com/mlavin/django-selectable.git@45b0dc8#egg=django-selectable
```

Comment:
Please use full commit hash. I don't know what would happen if there were multiple commits starting with `45b0dc8` in the future.